src/components/pages/Headmaster/HeadmasterProfile/ProfileForm.js

```diff
@@ -41,7 +43,7 @@ const dateFormatList = ['DD/MM/YYYY', 'DD/MM/YY'];
 
 const ProfileForm = props => {
   const [formData, setFormData] = useState(initialState);
-  const [value, setValue] = useState(1);
+  const [value, setValue] = useState('');
```

Comment:
Why this change in useState?

Reply:
@HeyMichelle because the Radio Input uses a string 'on' to indicate if its been checked.
I would suggest changing the name of that variable to something like checkBoxValue or something to make it quick and easy to understand in a future PR. But looks good to me.